PhysicsTools/Heppy/python/physicsobjects/Electron.py

```diff
@@ -5,6 +5,25 @@
 import sys
 from math import exp
 
+effective_areas = dict(
+    ea03 = [
+        (1.000, 0.1440),
+        (1.479, 0.1562),
+        (2.000, 0.1032),
+        (2.200, 0.0859),
+        (2.300, 0.1116),
+        (2.400, 0.1321),
+        (2.500, 0.1654)        
+        ]
+)
```

Comment:
for EA's, they're already in the LeptonAnalyzer, that also has also a proper switch for the different EA values depending on the MC era they're based on (e.g. spring16 vs summer16 vs fall17): https://github.com/CERN-PH-CMG/cmg-cmssw/blob/heppy_94X_dev/PhysicsTools/Heppy/python/analyzers/objects/LeptonAnalyzer.py#L361
I don't necessarily object at refactoring them to the electron object (or maybe to another file loaded by the electron object), but I *do* object at having them here as a single set of numbers and with no era and no reference to the source.